app/views/exercises/_results_card.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="card-result modal fade mx-5 p-5" id="commentModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <h1 class="text-center pt-5 mb-4">Congratulations!</h1>
```

Comment:
If you  put `modal-dialog` class here on the h1 it should start to work but it is missing some other things like the dismiss button and other bootstrap divs/classes